resources/nomad-jobs/sqlpad-connect.hcl

```diff
@@ -34,9 +34,17 @@ job "sqlpad" {
           }
         }
       }
+      check {
+        expose   = true
+        name     = "sqlpad-live"
+        type     = "http"
+        path     = "/"
```

Comment:
Are you sure that this request will return `200` and not `3xx` redirect somewhere?

Reply:
Yeah, it comes back healthy